yb-voyager/cmd/exportData.go

```diff
@@ -348,6 +347,36 @@ func getPGDumpSequencesAndValues() (map[*sqlname.SourceName]int64, error) {
 	return result, nil
 }
 
+func reportUnsupportedTables(finalTableList []*sqlname.SourceName) {
+	//report Partitions or case sensitive tables
+	var caseSensitiveTables []string
+	var partitionedTables []string
+	for _, table := range finalTableList {
+		if table.ObjectName.MinQuoted != table.ObjectName.Unquoted {
+			caseSensitiveTables = append(caseSensitiveTables, table.Qualified.MinQuoted)
+		} 
+		if source.DB().ParentTableOfPartition(table) == "" { //For root tables only
+			if len(source.DB().GetPartitions(table)) > 0 {
+				partitionedTables = append(partitionedTables, table.Qualified.MinQuoted)
+			}
+		} else {	
+			if source.TableList != "" { //If user has passed table list with leaf partition
```

Comment:
Do we really need this check? As long as it is a partition it is not supported.

Reply:
This was to make the partitions list efficient for users where if table-list is not provided then when we fetch the table list from the source, it also contains child partitions as well along with root partitions, in that case, we need not report about the child partitions only root table name is sufficient to be excluded in the list.
But now I realize that this will be not sufficient if a user is using only the exclude-table list with the root table, we will still have the table list from the source (which will eventually have the child partitions). 
Thanks, will fix it!